### PR TITLE
Take the last word in accoun.

### PR DESCRIPTION
**Pull request by Shakenbeer**
In the current version of the solution last word was omitted if it was followed be EOF. This input
q q q
qq qq
qqq qqq
produces 
Horizontal Histogram
  1:    ###
  2:    ##
  3:    #
but should be
Horizontal Histogram
  1:    ###
  2:    ##
  3:    ##

### CODE REVIEW

File: chapter_1/exercise_1_13/histogram.c [hunk 1]
```diff
@@ -43,6 +43,7 @@ int main(void)
 
         word_count_index = 0;
       }
+      if (c == EOF) break;
```
Comment (ohkimur) [inline chapter_1/exercise_1_13/histogram.c:46]: Please update this line to this syntax:
```c
if (c == EOF)
{
  break;
}
```

**Review by ohkimur — CHANGES_REQUESTED** [review]

**Comment by ohkimur** [comment]
@Shakenbeer Thank you very much for your contribution. I left in the review a request comment so that we keep the repo consistent.

File: chapter_1/exercise_1_13/histogram.c (hunk 1 above)
Comment (Shakenbeer) [reply chapter_1/exercise_1_13/histogram.c]: Done

**Review by ohkimur — APPROVED** [review]

File: chapter_1/exercise_1_13/histogram.c (hunk 1 above)
Comment (ohkimur) [reply chapter_1/exercise_1_13/histogram.c]: Great job! You're quick!

**Pull request merged**